Plugin/Catalog/Block/AbstractBlock.php

```diff
@@ -101,11 +102,13 @@ public function __construct(
         SearchEngine $searchEngineHelper,
         Logger $logger
     ) {
+        $this->loggerTraitConstruct(
+            $logger
+        );
         $this->categoryService = $categoryService;
         $this->paramResolver = $parameterResolver;
         $this->nostoCmpHelperData = $nostoCmpHelperData;
         $this->nostoHelperAccount = $nostoHelperAccount;
-        $this->logger = $logger;
```

Comment:
Remove `getLogger` method and rely only on the trait.  There are a few usages of it like: https://github.com/Nosto/nosto-magento2-cmp/blob/develop/Plugin/Catalog/Block/Toolbar.php#L167